### DISCUSSION TITLE
RP3 Receiver Diversity signal

### DISCUSSION THREAD

**Discussion by x4code** [ExpressLRS/ExpressLRS#2157] [Q&A]
Hello,
i own a radiomaster RP3 receiver (Antenna diversity)  and i think there is something is wrong with it.
everytime i connect to my radio ,always shows signal from Antenna:1 and RSSI_2. On RSSI_1 is something like -82. ive tried to block antenna and force to switch antennas, but no matter what i do, is always on the same antenna and signal on RSSI_2.

Any1 had this issue before?
Thank you

Regards,
Filipe

**Comment by rueckwaertsflieger** [comment]
For clarification: Are you saying, your telemetry page in the transmitter reports antenna 1 always active? There's 2 answers:
Antenna on antenna port 2 is defect, or
antenna switch is defect.
Not yet observed with my setups, I have 2 diversity receivers: matek and RP3.

**Reply by x4code** [reply]
Thanks for your Reply , really apreciate.
is there a way to test if its working properly?
can i just disconnect one antenna and see if it switchs?
yes i was watching on my telemetry page on radio.
Values are like this always:
RSS_1: -80   (but then shows as red)
RSS_2: -10 (right values)
Antenna: 1

on Your RP3 can you see getting signal to other antenna?
Thanks

**Reply by JyeSmith** [reply]
Yep, just remove a antenna and see if the rssi updates.

**Reply by x4code** [reply]
> Yep, just remove a antenna and see if the rssi updates.

it will not burn anything without RX antenna, right?

**Reply by JyeSmith** [reply]
The risk is already very low and I regularly dont use antennas when testing.

But with this rx the risk is pretty much zero.  TLM is sent through the antenna with the better reception, which will be the UFL with the antenna.

**Reply by x4code** [reply]
> The risk is already very low and I regularly dont use antennas when testing.
> 
> But with this rx the risk is pretty much zero. TLM is sent through the antenna with the better reception, which will be the UFL with the antenna.

will try that tomorrow, thanks a lot :)
Need to figure it out how to configure low RSSI alert on EdgeTX , hehe
